lib/from_erlang.ml

```diff
@@ -161,16 +161,30 @@ and extract_match_expr e =
        let (acc, v') = extract_match_expr' acc false v in
        (acc, F.ExprAssocExact (line, k', v'))
   in
-  let (es, _) = extract_match_expr' [] true e in
-  List.rev es
+  let (es, _) = extract_match_expr' [] true e in List.rev es
+
+(*
+let line_t_of_literal = function
+  | F.LitAtom (_line_t, name) -> _line_t
+  | LitInteger (_line_t, i) -> _line_t
+  | LitBigInt (_line_t, z) -> _line_t
+  | LitString (_line_t, s) -> _line_t
+
+(* [e1; e2; ...] という式の列を let _ = e1 in let _ = e2 ... in という１つの式にする *)
+let rec expr_of_exprs = function
+  | [] -> unit
+  | [e] -> e
+  | e :: es ->
+     Let ("_", e, expr_of_exprs es)
+*)
 
 let expr_of_atom_or_var = function
-  | F.AtomVarAtom (_line_t, a) -> Constant (Atom a)
-  | AtomVarVar (_line_t, v) -> Var v
+  | F.AtomVarAtom (_line_t, a) -> Constant (_line_t, Atom a)
+  | AtomVarVar (_line_t, v) -> Var (_line_t, v)
 
 let expr_of_integer_or_var = function
-  | F.IntegerVarInteger (_line_t, i) -> Constant (Number i)
-  | IntegerVarVar (_line_t, v) -> Var v
+  | F.IntegerVarInteger (_line_t, i) -> Constant (_line_t, Number i)
+  | IntegerVarVar (_line_t, v) -> Var (_line_t, v)
```

Comment:
nits: I think `_line_t` should be `line` because `_` prefix means "this is an unused variable" and `_t` prefix means "this is a type".

Reply:
Sorry, s/`_t` prefix/`_t` suffix/

Reply:
I fixed that point. Thanks.